pkgs/development/python-modules/pykde4/default.nix

```diff
@@ -0,0 +1,37 @@
+{ pyqt4,
+  stdenv, newScope, fetchurl, cmake, automoc4 }:
+
+let
+  callPackage = newScope self;
+  self = {
+    kdelibs = callPackage ./kdelibs.nix {};
+  };
+in with self; stdenv.mkDerivation rec {
+  version = "4.14.3";
+  name = "pykde4-${version}";
+
+  src = fetchurl {
+    url = "mirror://kde/stable/${version}/src/${name}-${version}.tar.xz";
+    sha256 = "1z40gnkyjlv6ds3cmpzvv99394rhmydr6rxx7qj33m83xnsxgfbz";
+  };
+
+  patches = [ ./dlfcn.patch ];
+
+  buildInputs = [
+    kdelibs
+  ];
+
+  nativeBuildInputs = [ cmake automoc4 ];
+
+  propagatedBuildInputs = [ pyqt4 ];
+
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
+    platforms = platforms.linux;
+    description = "Python bindings for KDE";
+    license = "GPL";
```

Comment:
Could we use a license or licenses from `lib.licenses`? `"GPL"` is ambiguous.

Reply:
@ttuegel Done, thanks!